From 72a3d2d72c39fd48f0a960a1b3c1e16e307421df Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mat=C3=ADas=20Hern=C3=A1ndez?= <matiashe@google.com>
Date: Fri, 6 Dec 2024 13:28:02 +0100
Subject: [PATCH] Remove excess parameter from EnqueueNotificationRunnable
 constructor call

Bug: 305695605
Fixes: 382077219
Test: atest NotificationManagerServiceTest
Flag: EXEMPT Test bugfix
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:31686e52218632a7b84e3d005094c74200050c16)
Merged-In: I677a5ad3a7497b87b933d7adc2917db02aafcfae
Change-Id: I677a5ad3a7497b87b933d7adc2917db02aafcfae
---
 .../server/notification/NotificationManagerServiceTest.java     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/tests/uiservicestests/src/com/android/server/notification/NotificationManagerServiceTest.java b/services/tests/uiservicestests/src/com/android/server/notification/NotificationManagerServiceTest.java
index fda54a46b4c8..7b6547ea5a35 100644
--- a/services/tests/uiservicestests/src/com/android/server/notification/NotificationManagerServiceTest.java
+++ b/services/tests/uiservicestests/src/com/android/server/notification/NotificationManagerServiceTest.java
@@ -14341,7 +14341,7 @@ public class NotificationManagerServiceTest extends UiServiceTestCase {
         assertThat(record.getNotification().getAllowlistToken()).isNull();
 
         mWorkerHandler.post(
-                mService.new EnqueueNotificationRunnable(mUserId, record, false, false,
+                mService.new EnqueueNotificationRunnable(mUserId, record, false,
                 mPostNotificationTrackerFactory.newTracker(null)));
         waitForIdle();
 
-- 
2.47.1.613.gc27f4b7a9f-goog

